HW10/script.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+dir="/home/tms/DevOps_TMS/$1"
+filename1="*.sh"
+filename2="*.png"
+
+find_filename1=`find $dir -type f -iname "$filename1"`
```

Comment:
Do find for both filetypes in one command

Reply:
Done!
![image](https://user-images.githubusercontent.com/44346029/113301921-34376880-9308-11eb-8dac-7b70785074e8.png)